spec/core-spec.md

```diff
@@ -254,13 +255,13 @@ Line Wavelength Grids
 The `wavelength_grid` mapping of a spectral line describes the wavelength quadrature provided as input into the program.
 The code is not required to use this wavelength grid, but it must indicate if it does not.
 
-The core spec requires support for the `"Linear"` and `"Tabulated"` wavelength grid types.
+The core spec requires support for the `"Linear"` and `"Table"` wavelength grid types. 
```

Comment:
Should clarify better what Linear means: a grid with constant separation built from `n_lambda` around (and including) the line rest wavelength. Should also define what happens when `n_lambda` is an even number (e.g. do not include the last point?). An alternative is to supply instead of the maximum wavelength range from line rest, and `n_lambda/2` instead?

Reply:
This is specified in the wavelength grid section further down, but could likely be cleaned up:

> delta_lambda: mapping of value (float) and unit (string) specifying the half-width of the transition from the line rest wavelength

Currently the approach is to accept an even n_lambda and simply generate `linspace(lambda0 - delta_lambda, lambda0 + delta_lambda, n_lambda)`, i.e. missing the rest wavelength if an even number of points is specified. It may be better to not accept this.

Reply:
Ahh, I see. I think that makes sense, just a range not necessarily including λ0.